db/schema.rb

```diff
@@ -630,7 +630,6 @@
     t.string  "status",            limit: 255
     t.string  "unityID",           limit: 255
     t.string  "signup_preference", limit: 255
-    t.string  "feedback",          limit: 255
   end
```

Comment:
This file(schema.rb) change is irrelevant with the issue fix, might have introduced because code difference in master and beta and team tried to merge.

Reply:
Should we edit and push it again?